android/build.gradle

```diff
@@ -15,13 +15,11 @@ buildscript {
     }
 
     dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"
-        classpath 'com.android.tools.build:gradle:8.5.0'
+        classpath 'com.android.tools.build:gradle:8.7.0-beta01'
         classpath "com.google.protobuf:protobuf-gradle-plugin:0.9.1"
-        classpath 'com.google.gms:google-services:4.4.0'
+        classpath 'com.google.gms:google-services:4.4.2'
```

Comment:
Why we are using `google-services` Can't we just remove it, Since we are not using firebase

Reply:
I think we just never removed it after we stopped using Firebase. Thanks for removing it